docs/content/preview/drivers-orms/go/yb-pgx.md

```diff
@@ -0,0 +1,224 @@
+      <i class="icon-postgres" aria-hidden="true"></i>
+      PGX Driver
+    </a>
+  </li>
+
+  <li >
+    <a href="/preview/drivers-orms/go/pq/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      PQ Driver
+    </a>
+  </li>
+
+</ul>
+
+The [YugabyteDB PGX driver](https://github.com/jackc/pgx/) is one of the most popular and actively maintained drivers for PostgreSQL. Use the driver to connect to YugabyteDB database to execute DMLs and DDLs using the PGX APIs. It also supports the standard `database/sql` package.
+
+## CRUD operations with PGX driver
+
+Learn how to establish a connection to YugabyteDB database and begin basic CRUD operations using the steps in the [Build an application](../../../quick-start/build-apps/go/ysql-yb-pgx) page under the Quick start section.
+
+The following sections break down the quick start example to demonstrate how to perform common tasks required for Go application development using the PGX driver.
+
+### Step 1: Import the driver package
+
+Import the YugabyteDB PGX driver package by adding the following import statement in your Go code:
+
+```go
+import (
+  "github.com/yugabyte/pgx/v4"
+)
+```
+
+[Optional] Learn how to use the `pgxpool` package in the [Using pgxpool API](/preview/reference/drivers/go/yb-pgx-reference/#using-pgxpool-api) section.
+
+### Step 2: Set up the database connection
+
+Go applications can connect to the YugabyteDB database using the `pgx.Connect()` and `pgxpool.Connect()` function. The `pgx` package includes all the common functions or structs required for working with YugabyteDB.
+
+Use the `pgx.Connect()` method or `pgxpool.Connect()` method to create a connection object for the YugabyteDB database. This can be used to perform DDLs and DMLs against the database.
+
+The following table describes the connection parameters required to connect to the YugabyteDB database with **Uniform load balancing**.
+
+| Parameter | Description | Default |
+| :---------- | :---------- | :------ |
+| user | user for connecting to the database | yugabyte
+| password | password for connecting to the database | yugabyte
+| host  | hostname of the YugabyteDB instance | localhost
+| port |  Listen port for YSQL | 5433
+| dbname | database name | yugabyte
+| load_balance | enables uniform load balancing | true
+
+The following is an example connection string for connecting to YugabyteDB with uniform load balancing.
+
+```sh
+postgres://username:password@localhost:5433/database_name?load_balance=true
+```
+
+The following is a code snippet for connecting to YugabyteDB using the connection parameters.
+
+```go
+baseUrl := fmt.Sprintf("postgres://%s:%s@%s:%d/%s",
+                    user, password, host, port, dbname)
+url := fmt.Sprintf("%s?load_balance=true", baseUrl)
+conn, err := pgx.Connect(context.Background(), url)
+```
+
+The following table describes the connection parameters required to connect to the YugabyteDB database with **Topology-aware load balancing**.
```

Comment:
No need to repeat the entire table for one additional parameter. Put it in the first table.